library/test/tests/hit-thread-limit.rs

```diff
@@ -0,0 +1,21 @@
+#![cfg(target_os = "linux")]
```

Comment:
I don't know should this be extended to be run on other platforms?

Reply:
You can limit it to single target using this syntax: https://github.com/rust-lang/rust/blob/0d97f7a96877a96015d70ece41ad08bb7af12377/src/test/ui/atomic-from-mut-not-available.rs#L2

Reply:
@mati865 Does that work here? This is not a UI test, but a Cargo [integration test](https://doc.rust-lang.org/nightly/cargo/reference/cargo-targets.html#integration-tests).

Reply:
Oh, sorry missed this fact!